docs/docs/concepts/plugins.mdx

```diff
@@ -423,9 +423,15 @@ that are applied to the extractor's [discovered catalog file](https://hub.meltan
 when the extractor is run using [`meltano run`](/reference/command-line-interface#run), [`meltano invoke`](/reference/command-line-interface#invoke), or [`meltano el`](/reference/command-line-interface#el).
 These rules are not applied when a catalog is [provided manually](#catalog-extra).
 
-A selection rule is comprised of an entity identifier that corresponds to a Singer stream's `tap_stream_id` value, and an attribute identifier that that corresponds to a Singer stream property name, separated by a period (`.`). Rules indicating that an entity or attribute should be excluded are prefixed with an exclamation mark (`!`). [Unix shell-style wildcards](<https://en.wikipedia.org/wiki/Glob_(programming)#Syntax>) can be used in entity and attribute identifiers to match multiple entities and/or attributes at once.
+The `select` extra defines **what properties within each stream** should be included in data extraction. It operates at the property level and determines which specific attributes of your data streams are selected for replication.
 
-Entity and attribute names can be discovered using [`meltano select --list --all <plugin>`](/reference/command-line-interface#select).
+A selection rule can be either:
+- **Stream-only pattern** (e.g., `"users"`): Selects the stream and all its properties, equivalent to `"users.*"`
```

Comment:
Before didn't this select the primary key(s)? I could be mistaken but if so it'd be a breaking change for some folks? Maybe not a huge deal

Reply:
@visch this still does select the primary keys, but yes it would select more by default now. I'd argue that's the more intuitive and expected behavior. If I select a table I'd likely be surprised just to see an ID and timestamp flow over. 

Not sure if this would warrant a full new v4 version, but I'd defer to @edgarrmondragon on that.

Reply:
I think this is the thing that made me reconsider https://github.com/meltano/meltano/pull/8810 and why it's still a draft.

It does the `inclusion: automatic` Singer metadata[^1] a bit redundant, unless I'm missing something.

[^1]: https://github.com/singer-io/getting-started/blob/master/docs/DISCOVERY_MODE.md#metadata